src/main/resources/db/changelog/changes/01-create-books-table.yaml

```diff
@@ -3,44 +3,44 @@ databaseChangeLog:
       id: create-books-table
       author: teract10s
       changes:
-      - createTable:
```

Comment:
pls make sure you've cleared Liquibase tables to perform this update


also try not to modify existing migrations in future